tests/garage/torch/optimizers/test_torch_conjugate_gradient_optimizer.py

```diff
@@ -0,0 +1,114 @@
+"""Tests for garage.torch.optimizers.conjugateGradientOptimizer."""
+import numpy as np
+import pytest
+import torch
+
+from garage.torch.optimizers.conjugate_gradient_optimizer import (
+    build_hessian_vector_product, conjugate_gradient)
+
+# pylint: disable=not-callable  #https://github.com/pytorch/pytorch/issues/24807  # noqa: E501
+
+
+class TestConjugateGradientOptimizer:
```

Comment:
note that pytest is perfectly happy running test functions outside of classes, as long as they are named `test_*`